Makefile

```diff
@@ -36,7 +36,7 @@ SRCS=$(shell \find bin lib tools -name '[^.]*.ml' -or -name '[^.]*.mli' -or -nam
 
 .PHONY: fmt
 fmt:
-	dune exec -- ocamlformat -i $(SRCS)
+	dune build @bin/fmt @lib/fmt @tools/fmt
```

Comment:
If we supported directories or globbing in `.ocamlformat-ignore` files, this would not be needed. I'd like to work with only dune, without magic parameters.
Should we improve that or is Dune going to have that feature ?
What do you think @emillon ?